.gitignore

```diff
@@ -60,3 +60,6 @@ Thumbs.db
 
 # Next.js
 .next
+
+# nx
+.nx/cache
```

Comment:
This is more of a nx17 thing, but somehow even at nx 16 these dirs are being used for me locally. Adding just for sanity.